### PR TITLE
warm-up module 00 exercise 02 i

### PR DESCRIPTION
**Pull request by danctorres**
Add src code for first point of module 00 exercise 02

Issue: #76

### CODE REVIEW

File: modules/00/e02/src/convert.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    std::string result = input;
+    for (char& c : result) {
+        c = std::tolower(c);
+    }
+    return result;
+}
+
+int main(int argc, char* argv[]) {
+    std::string command = argv[1];
+    std::string input = argv[2];
```
Comment (rui-rocha-42) [inline modules/00/e02/src/convert.cpp:23]: You're only considering the first word as input

File: modules/00/e02/src/convert.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+#include <string>
+#include <stdexcept>
+
+std::string convertToUpper(const std::string& input) {
+    std::string result = input;
+    for (char& c : result) {
+        c = std::toupper(c);
```
Comment (rui-rocha-42) [inline modules/00/e02/src/convert.cpp:8]: Implicit conversion from int to char. To avoid issues, you should do static_cast<char>(std::toupper(c));

**Review by jpalmeidaalves — CHANGES_REQUESTED** [review]
Although everything is working well you could use more modern classes and methods that  do the job in a more efficient way.

**Comment by danctorres** [comment]
Closing in favor of #80

**Pull request closed without merging**